src/main/groovy/de/dkfz/eilslabs/batcheuphoria/jobs/Job.groovy

```diff
@@ -209,6 +227,10 @@ class Job<J extends Job> {
         return tool
     }
 
+    String getToolScript() {
```

Comment:
I think you could remove most of the getters. They are auto generated by Groovy.